index.js

```diff
@@ -106,17 +133,22 @@ class LedgerBridgeKeyring extends EventEmitter {
         .then(async (_) => {
           const from = this.unlockedAccount
           const to = from + n
-          this.accounts = []
           for (let i = from; i < to; i++) {
+            const path = this._getPathForIndex(i)
             let address
             if (this._isBIP44()) {
-              const path = this._getPathForIndex(i)
               address = await this.unlock(path)
-              this.accountIndexes[ethUtil.toChecksumAddress(address)] = i
             } else {
               address = this._addressFromIndex(pathBase, i)
             }
-            this.accounts.push(address)
+            this.accountDetails[ethUtil.toChecksumAddress(address)] = {
+              bip44: this._isBIP44(),
+              hdPath: path,
+            }
+
+            if (!this.accounts.includes(address)) {
```

Comment:
Adding the same account more than once was not possible before, because the list of accounts was always cleared when adding a new account (See [L109](https://github.com/MetaMask/eth-ledger-bridge-keyring/pull/48/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346L109))

Reply:
Oh, of course :facepalm: Thanks, that makes sense.